Modules/atexitmodule.c

```diff
@@ -27,7 +27,10 @@ int
 PyUnstable_AtExit(PyInterpreterState *interp,
                   atexit_datacallbackfunc func, void *data)
 {
-    assert(interp == _PyInterpreterState_GET());
+    PyThreadState *tstate = _PyThreadState_GET();
+    _Py_EnsureTstateNotNULL(tstate);
```

Comment:
You can use PyInterpreterState_Get(), or PyThreadState_Get(), instead: it does the same.

Reply:
I think it's better to be explicit rather than do it with one of those.